core/shim/object.js

```diff
@@ -81,7 +81,7 @@ if (!Object.assign) {
             }
             for (s = 1, ss = arguments.length; s < ss; ++s) {
                 source = arguments[s];
-                keys = Object.keys(source);
+                keys = source ? Object.keys(source) : [];
```

Comment:
Better to avoid creating an empty array in a loop. Better to use a shared instance, or avoid creating an empty array altogether.

Reply:
Good catch! Fixed.

https://github.com/montagejs/montage/pull/2007/commits/a078d127e1e57bb29c47e3838d086677919a8ea9

Reply:
👍🏻